tests/PyzTest/Glue/Checkout/RestApi/Fixtures/ServicePointShipmentTypeCheckoutDataRestApiFixtures.php

```diff
@@ -10,12 +10,14 @@
 use ArrayObject;
 use Generated\Shared\Transfer\CountryTransfer;
 use Generated\Shared\Transfer\CustomerTransfer;
+use Generated\Shared\Transfer\ProductOfferStockTransfer;
```

Comment:
* Missed the changes for the file: `src/Pyz/Client/ShipmentTypeStorage/ShipmentTypeStorageConfig.php`

Reply:
Let's add it to be future proof)

Reply:
Added